### DISCUSSION TITLE
Help required - Unraid can't verify whether there is a new version of this docker anymore

### DISCUSSION THREAD

**Discussion by Guesch** [Koenkk/zigbee2mqtt#16231] [General]
Spoiler: There is one ;(

Not sure if I'm the only one impacted but I have no idea how to troubleshoot / resolve this issue: My docker version is 1.29.1 and when "Unraid docker" does an update check, the update status is "Not available" for this docker only.
As in, it works perfectly for all the other dockers I have by either telling me they are up-to-date or prompting me to update them.

Considering the issue is specific to this docker, I want to assume this is not on my end; but I have no idea why suddently Unraid would be unable to fetch update information from this github.
Anyone has a clue on how one could fix this issue?

Thank you!

**Comment by WernerRH21** [comment]
Same problem here.
zigbee2mqtt & netdata docker

**Comment by hanswerner-hw** [comment]
I had the same problem, docker on unraid

**Comment by Koenkk** [comment]
This will be fixed with the next z2m release (1 February) https://github.com/Koenkk/zigbee2mqtt/pull/16297
